### DISCUSSION TITLE
Duda

### DISCUSSION THREAD

**Discussion by Michelleeeer** [LaunchX-InnovaccionVirtual/CursoIntroPython#126] [Q&A] [answered]
Disculpen a ustedes ya se les habilito la sección de Estructuras de control a Manejo de errores, es que a mi no me aparece y dice que iba estar disponible desde ayer, pero no.

**Comment by DanielOe21** [comment, marked as answer]
Se indicó en las sesiones que las liberarían después, ya que aún faltan personas que se integrarán al programa y es con la intención de no dejarlos tan atrás, por mientras te sugiero que sigas practicando o incluso que busques en internet más ejercicios de programación para seguir mejorando!

PD: Hay que ver los lives / grabaciones 👀

PD2: Hay que ser un poco más descriptivos en las preguntas que se hagan, el título y el cuerpo del mensaje en lo personal me costó bastante entender a qué te referías.

PD3: Si tu pregunta queda contestada, no olvides marcarla como tal, además te sugiero que edites por lo menos el título por si algún otro compañero tiene la misma duda.

Salu✌️
